app/models/leader.rb

```diff
@@ -0,0 +1,10 @@
+class Leader < ApplicationRecord
```

Comment:
model looks good, potentially add documentation, or make sure it's easy to explain to them how to use it.